docs/admin/installation/hardware.rst

```diff
@@ -474,11 +500,10 @@ Server we recommend. To do so, select the "Boot and Install with the HWE
 Kernel" option in the boot menu for Ubuntu Server.
 
 .. _nuc11_recommendation:
-.. _intel-11th-gen-nuc:
```

Comment:
Why was this anchor removed? It was intentionally added in 5f098d78fb98ac8c9c3a021aad0f5730fe52204e.

Reply:
I've standardized on the "nuc[x]_recommendation" format. This will break some historical anchors, but given the amount of time that's passed that's IMO fine.